pkg/domain/infra/abi/play.go

```diff
@@ -360,7 +360,7 @@ func (ic *ContainerEngine) playKubePod(ctx context.Context, podName string, podY
 	}
 
 	if len(options.Networks) > 0 {
-		ns, networks, netOpts, err := specgen.ParseNetworkFlag(options.Networks)
+		ns, networks, netOpts, err := specgen.ParseNetworkFlag(options.Networks, false)
```

Comment:
this should not be forced to false we need to check it here as well, use ic.Libpod.Network().NetworkInspect()

Reply:
Changed.